services/upload/src/main/java/app/coronawarn/server/services/federation/upload/payload/UploadPayload.java

```diff
@@ -42,6 +41,7 @@
   private List<FederationUploadKey> originalKeys;
   private String batchSignature;
   private String batchTag;
+  private List<DiagnosisKey> orderedKeys;
```

Comment:
Not sure if it's needed to store a copy of the keys in the upload payload. Rather whenever we need the ordered list we can on demand sort the one in the batch. This however is not crucial and can be dealt with in a later refactoring step.